src/internal.c

```diff
@@ -1392,6 +1392,15 @@ int InitSSL_Ctx(WOLFSSL_CTX* ctx, WOLFSSL_METHOD* method, void* heap)
                 #error Micrium port does not support DTLS session export yet
             #endif
         #endif
+    #elif defined UIP
```

Comment:
Can we use a define like `WOLFSSL_UIP`? If not then can we have the `UIP` define set `WOLFSSL_UIP` in settings.h? Something like:
 
```
#if defined(UIP) && !defined(WOLFSSL_UIP)
#define WOLFSSL_UIP
#endif
```

Also to be consistent with our coding pattern can you use syntax with parens around it like: `#elif defined(WOLFSSL_UIP)`.